exercises/01.compartmentalisation/cheri.js

```diff
@@ -0,0 +1,98 @@
+// FFI Imports
+// Each function imported from the host environment needs to be assigned to a
+// global like this and identified by a constant that the resolver in the C/C++
+// code will understand.
+
+
+/**
+ * Log function, writes all arguments to the UART.
+ */
+export const print = vmImport(1);
```

Comment:
Document that these constants are elements of the `Exports` `enum` down in `js.cc`?